### PR TITLE
Cleanup --from-tasty blacklists

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: compiler/test/dotty/tools/TestSources.scala [hunk 1]
```diff
@@ -43,7 +43,9 @@ object TestSources {
       .filter(_.nonEmpty)
       .toList
 
-    assert(list.nonEmpty)
+    if (list.isEmpty)
+      println(s"$path is empty")
```
Comment (allanrenucci) [inline compiler/test/dotty/tools/TestSources.scala]: Why this change?

Comment (nicolasstucki) [reply compiler/test/dotty/tools/TestSources.scala]: To be able to remove all blacklisted item and run the tests to see which one actually fail.
